src/main/java/org/geysermc/resources/state/type/wall/PostBitMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.geysermc.resources.state.type.wall;
+
+import org.geysermc.resources.Pair;
+import org.geysermc.resources.state.StateMapper;
+import org.geysermc.resources.state.StateRemapper;
+
+@StateRemapper(value = "up", blockRegex = ".*_wall.?$")
+public class PostBitMapper extends StateMapper<Boolean> {

```

Comment:
Needs to be named to WallUpMapper. Naming convention is `(block)(java state)Mapper`.